package.json

```diff
@@ -15,15 +15,7 @@
     "hoodie-plugin-react-native-server": "^0.5.0"
   },
   "hoodie": {
-    "address": "127.0.0.1",
-    "port": 8080,
-    "data": ".hoodie",
-    "public": "public",
-    "dbUrl": "admin:admin@http://127.0.0.1:5984/",
-    "dbAdapter": "pouchdb-adapter-fs",
-    "inMemory": false,
```

Comment:
it’s either `dbUrl` or `inMemory` or `dbAdapter`. `dbUrl` automatically sets the adapter to `pouchdb-adapter-http` and `inMemory` sets it to `pouchdb-adapter-memory`

Also you usually wouldn’t set `dbUrl` in thepackage.json but with an environment argument as it usually contains sensitive credentials